Client build error: fatal error: boost/optional.hpp: No such file or directory

From [Deploy: DEV or STAGING #18](https://github.com/slateci/slate-client-server/actions/runs/3525716745/jobs/5912819977):
```shell
  export VERSION_OVERRIDE=1.0.32-pre.20221122-173350
  make
  shell: sh -e {0}
  env:
    RELEASE_VERSION: 1.0.32-pre.20221122-173350
Scanning dependencies of target embed_version
[  3%] Generating client_version.h, client_version.h_
[  3%] Built target embed_version
Scanning dependencies of target slate
[  7%] Building CXX object CMakeFiles/slate.dir/src/client/slate_client.cpp.o
[ 11%] Building CXX object CMakeFiles/slate.dir/src/client/Client.cpp.o
/__w/slate-client-server/slate-client-server/checkout/src/client/Client.cpp:2404:4: warning: #warning TODO: container name should be URL encoded [-Wcpp]
 2404 |   #warning TODO: container name should be URL encoded
      |    ^~~~~~~
[ 15%] Building CXX object CMakeFiles/slate.dir/src/client/ClusterRegistration.cpp.o
[ 19%] Building CXX object CMakeFiles/slate.dir/src/client/Completion.cpp.o
[ 23%] Building CXX object CMakeFiles/slate.dir/src/client/SecretLoading.cpp.o
[ 26%] Building CXX object CMakeFiles/slate.dir/src/client/cluster_components/FederationRBAC.cpp.o
[ 30%] Building CXX object CMakeFiles/slate.dir/src/client/cluster_components/IngressController.cpp.o
[ 34%] Building CXX object CMakeFiles/slate.dir/src/client/cluster_components/PrometheusMonitoring.cpp.o
[ 38%] Building CXX object CMakeFiles/slate.dir/src/Archive.cpp.o
[ 42%] Building CXX object CMakeFiles/slate.dir/src/FileHandle.cpp.o
[ 46%] Building CXX object CMakeFiles/slate.dir/src/FileSystem.cpp.o
[ 50%] Building CXX object CMakeFiles/slate.dir/src/HTTPRequests.cpp.o
[ 53%] Building CXX object CMakeFiles/slate.dir/src/KubeInterface.cpp.o
In file included from /__w/slate-client-server/slate-client-server/checkout/include/crow.h:4,
                 from /__w/slate-client-server/slate-client-server/checkout/include/Telemetry.h:9,
                 from /__w/slate-client-server/slate-client-server/checkout/src/KubeInterface.cpp:9:
/__w/slate-client-server/slate-client-server/checkout/include/crow/query_string.h:9:10: fatal error: boost/optional.hpp: No such file or directory
    9 | #include <boost/optional.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/slate.dir/build.make:219: CMakeFiles/slate.dir/src/KubeInterface.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:78: CMakeFiles/slate.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
Error: Process completed with exit code 2.
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -191,7 +191,6 @@ LIST(APPEND STRESS_TEST_SOURCES
         ${CMAKE_SOURCE_DIR}/src/KubeInterface.cpp
         ${CMAKE_SOURCE_DIR}/src/Process.cpp
         ${CMAKE_SOURCE_DIR}/src/Utilities.cpp
-        ${CMAKE_SOURCE_DIR}/src/Telemetry.cpp
         )
 SET(STRESS_TEST_COMPILE_OPTIONS
         -DSLATE_CLIENT
@@ -206,7 +205,6 @@ target_include_directories (slate-stress-test
         ${CURL_INCLUDE_DIRS}
         ${LIBCRYPTO_CFLAGS}
         ${SSL_CFLAGS}
-        ${OPENTELEMETRY_CPP_INCLUDE_DIRS}
         $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}>
         )
 target_link_libraries(slate-stress-test
@@ -216,8 +214,6 @@ target_link_libraries(slate-stress-test
         ${LIBCRYPTO_LDFLAGS}
         z
         pthread
-        ${OPENTELEMETRY_CPP_LIBRARIES}
-        nlohmann_json::nlohmann_json
         )
 target_compile_options(slate-stress-test PRIVATE -DRAPIDJSON_HAS_STDSTRING -Os -std=c++11)
 
@@ -328,8 +324,8 @@ if(BUILD_SERVER)
             src/Archive.cpp
             src/FileHandle.cpp
             src/FileSystem.cpp
-            src/Process.cpp
-             src/Telemetry.cpp)
+            src/Process.cpp)
+#             src/Telemetry.cpp)
     target_include_directories (slate-test-database-server
             PUBLIC
             $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
@@ -341,7 +337,7 @@ if(BUILD_SERVER)
             ${CURL_INCLUDE_DIRS}
             ${LIBCRYPTO_CFLAGS}
             ${SSL_CFLAGS}
-            ${OPENTELEMETRY_CPP_INCLUDE_DIRS}
+#            ${OPENTELEMETRY_CPP_INCLUDE_DIRS}
             )
     target_link_libraries(slate-test-database-server
             PUBLIC
@@ -351,7 +347,7 @@ if(BUILD_SERVER)
             ${Boost_LIBRARIES}
             ${ZLIB_LIBRARIES}
             pthread
-            ${OPENTELEMETRY_CPP_LIBRARIES}
+#            ${OPENTELEMETRY_CPP_LIBRARIES}
             )
 
     add_executable(slate-test-helm-server
@@ -368,7 +364,7 @@ if(BUILD_SERVER)
             ${CURL_INCLUDE_DIRS}
             ${LIBCRYPTO_CFLAGS}
             ${SSL_CFLAGS}
-            ${OPENTELEMETRY_CPP_INCLUDE_DIRS}
+#            ${OPENTELEMETRY_CPP_INCLUDE_DIRS}
             )
     target_compile_options(slate-test-helm-server PRIVATE ${YAMLCPP_CFLAGS} )
     target_link_libraries(slate-test-helm-server
@@ -380,7 +376,7 @@ if(BUILD_SERVER)
             ${Boost_LIBRARIES}
             ${ZLIB_LIBRARIES}
             pthread
-            ${OPENTELEMETRY_CPP_LIBRARIES}
+#            ${OPENTELEMETRY_CPP_LIBRARIES}
             )
 
     file(MAKE_DIRECTORY "${CMAKE_BINARY_DIR}/tests")

--- a/src/KubeInterface.cpp
+++ b/src/KubeInterface.cpp
@@ -3,10 +3,13 @@
 #include <memory>
 #include <string>
 #include <iostream>
+#include <sstream>
 
 #include "Utilities.h"
 #include "FileHandle.h"
+#ifdef SLATE_SERVER
 #include "Telemetry.h"
+#endif
 
 namespace kubernetes{
 	

